Source/WTF/wtf/spi/darwin/XPCSPI.h

```diff
@@ -171,7 +171,7 @@ extern const struct _xpc_type_s _xpc_type_endpoint;
 extern const struct _xpc_type_s _xpc_type_error;
 extern const struct _xpc_type_s _xpc_type_string;
 
-xpc_object_t xpc_array_create(const xpc_object_t*, size_t count);
```

Comment:
Do not remove this (or xpc_dictionary_create) from the XPCSPI.h header. There are legit use cases for this function. Just add the `create_empty` variants to this header.